src/js/containers/search/SearchContainer.jsx

```diff
@@ -184,6 +184,15 @@ const SearchContainer = ({ history }) => {
         }
     }, [appliedFilters]);
 
+    useEffect(() => {
+        if (generateHashInFlight) {
+            setGenerateHashInFlight(false);
+        }
+        if (downloadInFlight) {
+            setDownloadInFlight(false);
+        }
+    }, [generateHashInFlight, downloadInFlight]);
```

Comment:
we don't want this because it's going to set our loading state to false any time it changes from false to true. Check out the useEffect documentation to see what happens here.